src/app/app.component.html

```diff
@@ -1,31 +1,9 @@
 <mat-sidenav-container class="container">
   <mat-sidenav class="container" mode="side" [(opened)]="opened">
+    <span><img src="assets/images/logo.png" width="160px" alt="Clean CaDET"></span>
```

Comment:
This <img> can be a part of the cc-navbar component. No need to separate it.

Regarding the cc-navbar (shown below) I have a few suggestions for improvements:
- The logo should be centered like the buttons
- The buttons can have smaller text, maybe just 1 step larger than the default for the IE and AE texts.
- The navbar should be more narrow, somewhere along 60% of what it is now.
- The popup menu for the units should be themed like the app. We should find a way to consistently apply the theme, instead of copy-pasting "background-color: #141414;  color: white;" everywhere.
- Maybe the home button can be removed, and instead clicking on the logo leads to home?

![image](https://user-images.githubusercontent.com/7092212/144803998-e9d92215-3b4e-47d1-99ae-e84e45fd2f2f.png)